frontend/src/styles/base/_spacing.scss

```diff
@@ -5,15 +5,15 @@
 // ----------------------------------------------------------------------
   // #Spacing Values
 // ----------------------------------------------------------------------
-$space-xxxs: 0.25em;
-$space-xxs: 0.375em;
-$space-xs: 0.5em;
-$space-sm: 0.75em;
-$space-md: 1.25em;
-$space-lg: 2em;
-$space-xl: 3.25em;
-$space-xxl: 5.25em;
-$space-xxxl: 8.5em;
+$space-xxxs: 20px;
+$space-xxs: 40px;
+$space-xs: 60px;
+$space-sm: 80px;
+$space-md: 100px;
+$space-lg: 160px;
+$space-xl: 200px;
+$space-xxl: 240px;
+$space-xxxl: 280px;
```

Comment:
I'm worried about this when this transfers to mobile. In one sense that keeping the em around means that the spacing is proportional to the font size which will change based on the device/browser. Thoughts? You could always make space-xxxs for example 1.25em which in a desktop browser be 20pts.

Reply:
ok I will change them into em